src/Native/System.Native/pal_console.cpp

```diff
@@ -458,7 +458,9 @@ static bool InitializeSignalHandling()
     }
 
     // Finally, register our signal handlers
-    struct sigaction newAction = { .sa_flags = SA_RESTART | SA_SIGINFO };
+    struct sigaction newAction;
```

Comment:
I will initialise the other cases with the default values as well. We need to keep the code for System.Native clang and gcc compatible and that will mean more lines of code.